src/components/scenes/Fio/FioAddressListScene.tsx

```diff
@@ -144,24 +144,21 @@ export class FioAddressList extends React.Component<Props, LocalState> {
                 />
               ))}
             </View>
-            <Fade visible={loading && !initLoading}>
+            <EdgeAnim visible={loading && !initLoading} enter={{ type: 'fadeIn' }} exit={{ type: 'fadeOut' }}>
               <ActivityIndicator color={theme.iconTappable} style={styles.loading} size="large" />
-            </Fade>
+            </EdgeAnim>
           </ScrollView>
-
-          <View>
-            <ClickableText marginRem={[1, 1, 0]} onPress={() => navigation.navigate('fioAddressRegister', {})}>
-              <View style={styles.actionButton}>
-                <Fontello name="register-new-fio-icon" style={styles.actionIcon} color={theme.iconTappable} size={theme.rem(1)} />
-                <EdgeText style={styles.buttonText}>{lstrings.fio_address_list_screen_button_register}</EdgeText>
-              </View>
-            </ClickableText>
-            <ClickableText marginRem={[0, 1, 2, 1]} onPress={() => navigation.navigate('fioDomainRegister', {})}>
-              <View style={styles.actionButton}>
-                <Fontello name="register-custom-fio" style={styles.actionIcon} color={theme.iconTappable} size={theme.rem(1)} />
-                <EdgeText style={styles.buttonText}>{lstrings.fio_address_list_domain_register}</EdgeText>
-              </View>
-            </ClickableText>
+          <View style={styles.buttons}>
```

Comment:
`sceneMargin` gives it way too much bottom margin. But I need margin on the left and right which I can't just pass to `ButtonsView`

Reply:
The bottom margin was to aid in tap accuracy and research about user expectations of being able to scroll an element higher into an easier to tap area of the screen. I don't have the research paper on the study handy, but the findings are that the bottom-most edge of the screen actually has worse tap accuracy than a bit higher up for the one-handed grip. 

I see that this case is a bit different since the buttons aren't part of the scroll. I'll take note of this "sticky scene footer" case to discuss for 4.1 design.